CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 14.0.0
+
+#### Breaking changes - [#525](https://github.com/openfisca/openfisca-core/pull/525)
+
+* Rename Variable attribute `url` to `reference`
+  - As previously, this attribute can be a string, or a list of strings.
+  - Variable attribute `law_reference` is removed.
+  - `TaxBenefitSystem.reference` renamed to `TaxBenefitSystem.reference_tbs`.
```

Comment:
My point was to keep `reference` as an attribute for `TaxBenefitSystem` since their `reference` is natural for two reasons:
 - the reference is also a TaxBenefitSystem
 - `reference/reform` is very clear dichotomy to me (`base` calls for `extension` and `original` calls for `copy`) 
   
Extension of the `reference` to the Variable as `reference_variable` or `reference` is on the same vein.

I am ok to use `source` or `legislative_reference`  instead of  `url`.